CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to leaderboard-backend
+We appreciate your help!
+
+## Before filing an issue
+If you are unsure whether you have found a bug, please consider asking in [our discord](https://discord.gg/TZvfau25Vb) first.
+
+Similarly, if you have a question about a potential feature, [the discord](https://discord.gg/TZvfau25Vb) can be a fantastic resource for first comments.
+
+## Filing issues
+Filing issues is as simple as going to [the issue tracker](https://github.com/speedrun-website/leaderboard-backend/issues), and adding an issue using one of the below templates.
+
+### Feature Request / Task
+```
+{Feature Request/Task}: {short description}
+---
+{Detailed description}
+
+### Affected Functionality
+{Any known functionality impacted, or Unknown if further research needs to be done}
+
+### Other Relevant Issues
+{Links to Relevant Issues}
+```
+
+### Bugs
+```
+Bug: {short description}
+---
+{Summary of bug}
+
+### Step(s) to Reproduce
+{Numbered list of step(s)}
+
+### Expected Result
+{Summary of expected result}
+
+### Actual Outcome
+{Description of actual outcome}
+```
+
+## Contributing code
+### Example code contribution flow
+1. Make a fork of this repo.
+1. Name a branch on your fork something descriptive for this change (eg. `UpdateMakefile`).
```

Comment:
Could add a "Optional but preferred: Reference the issue or task you are working on in the branch name"